pkgs/tools/security/qdigidoc/default.nix

```diff
@@ -56,6 +53,14 @@ mkDerivation rec {
     qtsvg
   ];
 
+  cmakeFlags = [
+    # If not provided before the build, qdigidoc tries to download a TSL list during the build.
+    # We pass it in via TSL_URL, fetched from https://ec.europa.eu/tools/lotl/eu-lotl.xml.
+    "-DTSL_URL=file://${./eu-lotl.xml}"
+    # also expects config.{pub,rsa}
```

Comment:
That comment does not make clear how CMake itself substitues the suffix, i.e. expects all `config.*` with the same basename in one directory in order to fetch them all.

https://github.com/openbsd/ports/blob/8f725c7cb1dfa3238f1f220fe5c7e20bfe8bc928/security/qdigidoc4/Makefile#L28

Reply:
Done